crates/topos/tests/utils.rs

```diff
@@ -37,6 +40,8 @@ pub async fn setup_polygon_edge(path: &str) -> String {
     installation_path.to_str().unwrap().to_string()
 }
 
+// Have to allow dead_code because clippy doesn't recognize it is being used in the tests
+#[allow(dead_code)]
```

Comment:
does using `#[cfg(test)]` could work?

Reply:
Hmm. How can the compiler think it's dead code if it's `pub`? I assume the module is `pub` too?

Reply:
No, it didn't... I can try with other flags or dig deeper into why!